data/guides.yml

```diff
@@ -10,8 +10,10 @@
   chapters:
     - title: "Installing Ember"
       url: "index"
-    - title: "Glossary"
-      url: "glossary"
+    - title: "Background Concepts"
```

Comment:
I'd use "Foundation" or "Key background concepts" here. If you're writing this for new devs, they **must** grasp these or they'll have a bad time.